config/locales/ca_info.yml

```diff
@@ -1,5 +0,0 @@
-ca:
-  assembly_info: Són espais de trobada entre Administració i ciutadania per a dur a terme debats regulars a l’entorn de polítiques sectorials.
```

Comment:
We didn't talk offline that these translations had to change their key? because at the time of executing the Populate task, will they take these descriptions instead the default ones?

Reply:
Done. Thanks